frontend/src/Student/components/PhotoSubmissionForm.js

```diff
@@ -161,11 +161,7 @@ class PhotoSubmissionForm extends Component {
     }
 
     // calculate whether the user is beyond their single submissions
-    const numSingleEntries = this.props.data.show.entries.reduce(
-      // if 'student' is non-null, this is a single submission
-      (n, entry) => (entry.student ? n + 1 : n),
-      0
```

Comment:
What were they trying to do here?

Reply:
They were originally basing the disitinction between groups and single submissions based on whether there was an owner of the submission. Now its been changed to check if theres a group in the submission which makes a little more sense.